lib/puppet.rb

```diff
@@ -74,7 +74,7 @@ class << self
       end
 
       if locale_path
-        if Gem.loaded_specs['gettext-setup'].version < Gem::Version.new('0.8')
+        if GettextSetup.method(:initialize).parameters.count == 1
```

Comment:
If we're searching for the number of parameters, we can access that with `:arity`
ie
```
Foo.method(:bar).arity
=> 2
```